src/docfx/lib/JsonUtility.cs

```diff
@@ -346,7 +357,7 @@ private static bool IsNullOrUndefined(this JToken token)
                 (token.Type == JTokenType.Undefined);
         }
 
-        private static void FindNulls(this JToken token, List<Error> errors, List<JToken> nullNodes)
+        private static void RemoveNullsCore(JToken token, List<Error> errors, List<JToken> nullNodes, List<JToken> removeNodes)
```

Comment:
>private static void RemoveNullsCore(JToken token, List<Error> errors, Li [](start = 8, length = 72)

can this be a local function?

Reply:
If could be, I was just trying to keep the existing convention here